runtime/Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/centrifuge/centrifuge-chain"
 
 [dependencies]
 # external pallet
-substrate-pallet-multi-account = { version = "2.0.0-alpha.1", git = "https://github.com/centrifuge/substrate-pallet-multi-account", default-features = false }
+substrate-pallet-multi-account = { version = "2.0.0-alpha.3", git = "https://github.com/mikiquantum/substrate-pallet-multi-account", rev = "976ee2504281ce05ed26830491b178170ae8e1fa", default-features = false }
```

Comment:
temporary repo until https://github.com/centrifuge/substrate-pallet-multi-account/pull/3 is merged

Reply:
Can be used now.